Jenkinsfile

```diff
@@ -23,6 +32,26 @@ def build_dgl() {
   }
 }
 
+def build_dgl_win64() {
+  /* Assuming that Windows slaves are already configured with MSBuild VS2017,
+   * CMake and Python/pip/setuptools etc. */
+  bat "DEL /S /Q build"
+  bat "DEL /S /Q _download"
+  dir ("build") {
+    bat """
+    CALL "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Auxiliary\\Build\\vcvars64.bat"
+    cmake -DCMAKE_CXX_FLAGS="/DDGL_EXPORTS" -DCMAKE_CONFIGURATION_TYPES="Release" .. -G "Visual Studio 15 2017 Win64"
+    msbuild dgl.sln
+    COPY Release\\dgl.dll .
+    """
+  }
+  dir ("python") {
+    bat "DEL /S /Q build *.egg-info dist"
+    bat "pip3 uninstall -y dgl || (call )"  // ignore error
+    bat "python setup.py install --user"
+  }
+}
```

Comment:
Let's put these in a script file under `tests/scripts`: `task_build_dgl.sh` and `task_build_dgl.bat`.